src/game_config_game.cpp

```diff
@@ -105,6 +106,11 @@ void Game_ConfigGame::LoadFromArgs(CmdlineParser& cp) {
 					patch_key_patch.Set(true);
 				} else if (v == "rpg2k3-cmds" || v == "rpg2k3-commands") {
 					patch_rpg2k3_commands.Set(true);
+				} else if (v.rfind("direct-menu=") == 0) {
+					int num = static_cast<int>(std::strtol(v.substr(12).c_str(), nullptr, 0));
+					if (num > 0) {
+						patch_direct_menu.Set(num);
```

Comment:
That looks pretty ugly though not your fault. Maybe is better when we change the entire command line from 
``--patch PATCH1 PATCH2 PATCH3``

to ``--patch-PATCH1`` e.g. ``--patch-maniac``. Then ``--patch-direct-menu 123`` would work :thinking: 

But this can be changed later...